src/globals.css

```diff
@@ -173,6 +174,18 @@
     --orange-900: hsl(var(--color-orange-900));
     --shadow-sidebar: 0px 33px 32px -16px rgba(0, 0, 0, 0.1),
         0px 0px 16px 4px rgba(0, 0, 0, 0.04);
+
+    /* Toastify */
+    --toastify-toast-min-height: 48px;
+    --toastify-toast-bd-radius: 8px;
+    --toastify-text-color-light: #404446;
```

Comment:
please don't use arbitrary colors, #404446 === gray-800

Reply:
Added fix in: [ce2ecff](https://github.com/powerhouse-inc/design-system/pull/222/commits/ce2ecff1f34e4d54e28ed6ec9ffccb2adcf97104)